userland/examples/nrf51_aes/aes.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <tock.h>
```

Comment:
Since you're already making some changes, the standard we've adopted for headers is:

```c
#pragma once

#include <tock.h>

#ifdef __cplusplus
extern "C" {
#endif
```

That is, headers go outside of the `extern C` and will specify their own `extern C`'s if needed